INPTP_AppForFixing/MainForm.cs

```diff
@@ -167,5 +158,28 @@ private void btnEmplEdit_Click(object sender, EventArgs e)
                 showWarning("First you must select a boss and the employee which is about to be altered!");
             }
         }
+
+        private void MainForm_Load(object sender, EventArgs e)
+        {
+
+        }
+
+        private void btnGenerateSampleData_Click(object sender, EventArgs e)
```

Comment:
this implementation will not generate nested hierarchies... 
- boss
- - boss
- - - employee